### ISSUE TITLE
[AMENDMENT] Encrypted Messaging

### ISSUE THREAD

**Issue by magical-heyrovsky** [Lissy93/awesome-privacy#133]
### Title

Threema

### Amendments

In the [Encrypted Messaging](https://github.com/Lissy93/awesome-privacy#encrypted-messaging) it says

> Many messaging apps claim to be secure, but if they are not open source, then this cannot be verified - and they should not be trusted. This applies to Telegram, Threema, ...

But Threema is FOSS (https://github.com/threema-ch) and has been for quite some time now. For Android there is even a additional [Threema Libre](https://threema.ch/en/blog/posts/threema-libre) version (without proprietary libraries) that can be installed via the F-Droid store.

### Would you like to submit a PR?

No.

### Please tick the boxes

- [X] You have filled out this form accurately, and to the best of your knowledge
- [X] You have indicated weather or not you are associated with the project the amendment refers to
- [X] A similar submission has not already [been opened](https://github.com/Lissy93/awesome-privacy/issues?q=is%3Aissue) for this software/ service
- [X] You agree to the [code of conduct](https://github.com/Lissy93/awesome-privacy/blob/main/.github/CODE_OF_CONDUCT.md)

**Comment by ltGuillaume** [comment]
1. Not the server part
1. A recent blog stated they had only recently introduced (optional) perfect forward secrecy, which seemed rather odd to me at least.

**Comment by ltGuillaume** [comment]
1. Yes, that's sound reasoning, thx for elaborating. 
1. Yeah, while the issue was that people already assumed that Threema had PFE implemented (which basically shows how complicated/diverse e2ee implementations are), I guess adding it now at least is a positive. I don't use it, but I do wonder why they made it optional (I have only read a press release so far).

**Comment by ltguillaume** [comment]
@lissy93 I don't think this should be discarded, as Threema is still mentioned as "insecure" and I do think @magical-heyrovsky has a point.

If it's about the server part not being open source, one could say the same about Signal (at least parts of their server software are not open source).

**Comment by lissy93** [comment]
> > Not the server part

> Doesn't matter. E2EE can still be verified

For message content, yes. For metadata, no.
But yeah, slightly moot point, since no way to verify they're not running a modified/back-doored version.

---

Anyway, I've got a PR ready in #718 - let me know if you've got any thoughts @magical-heyrovsky :)
